### ISSUE TITLE
Sidebar closes when you click in an annotation editor (can't annotate)

### ISSUE THREAD

**Issue by seanh** [hypothesis/via#116]
1. Go to a page such as this one: https://via.hypothes.is/http://www.independent.co.uk/news/world/middle-east/iran-earthquake-latest-news-baby-found-alive-rubble-rescue-hunt-survivors-iraq-border-a8058261.html

2. Select some text and click _Annotate_

3. Sidebar opens, but annotation editor is not focused (it doesn't have the keyboard focus)

4. Click in the annotation editor and the sidebar closes. Every time you open the sidebar again and try to click in the annotation editor to give it the keyboard focus, the sidebar closes. You can't create an annotation.

It's a little hard to see in this GIF because there are multiple separate issues also present (multiple instances of the sidebar appearing in the page, the sidebar UI being shifted off-screen to the right so that you can't re-open the sidebar) but when I create an annotation the editor isn't focused and when I click in the editor the sidebar closes:

![peek 2017-11-16 16-49](https://user-images.githubusercontent.com/22498/32903986-47035842-caee-11e7-8824-762a257a6cf2.gif)

Possible related to this issue: https://github.com/hypothesis/client/issues/280

**Comment by seanh** [comment]
(Actually clicking _anywhere_ in the sidebar closes it.)

**Comment by robertknight** [comment]
That URL no longer functions and I haven't been able to find where it moved to or get an archived copy, so I'm closing this issue.
